### ISSUE TITLE
UI Quirks Megathread

### ISSUE THREAD

**Issue by KazWolfe** [KazWolfe/CollectorsAnxiety#5]
Some various quirks of the current plugin UI that should be looked at at some point.

- White-on-Yellow coloring for the progress bar completely spits on WCAG guidelines for contrast.
- Empty checkbox indicator may be confusing to some users as it's a non-interactable element.
    - Replace this with X and checkmark icons instead maybe?
- Right-click on checkbox is extremely hidden, causing users to miss useful functionality.
- Right-click on checkbox should be expanded to cover the entire row if possible. 
- Font scaling can cause column headers to become clipped.
- No ability to filter by badges (is marketboardable) is a bit frustrating to find things
